scripts/pr-vars.source.sh

```diff
@@ -6,7 +6,6 @@ export script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd
 pr_number=${PR_NUMBER:-44}
 export DEPLOYMENT_NAME="app-autoscaler-${pr_number}"
 export SERVICE_BROKER_NAME="app-autoscaler-${pr_number}servicebroker"
-export SERVICE_NAME="autoscaler-${pr_number}"
```

Comment:
hmm service name and deployment name are different for backward compat with the older acceptance test runs. 
I guess we can revisit it and check that we are not going to break clients that run it outside of our pipeline.
It think this would be a `breaking change`

Reply:
if there are clients using the pipeline yes we would break their pipeline, we can check this.